README.md

```diff
@@ -1,16 +1,49 @@
 # README
 
-TypeScript implementation for LIonWeb standards - currently: the `lioncore` meta-metamodel in [`src/m3/types.ts`](./src/m3/types.ts).
+TypeScript implementation for LIonWeb standards - currently: the LIonCore meta-metamodel (M3) in [`src/m3/types.ts`](src/m3/types.ts).
 
 
 ## Building
 
-Run [`watch-check-meta-circularity.sh`](./watch-check-meta-circularity.sh) to generate a PlantUML file [`diagrams/metametamodel-gen.puml`](./plantUML/metametamodel-gen.puml) from the meta-circular definition of `lioncore` in [`src/m3/test/self-definition.ts`](./src/m3/test/self-definition.ts).
-This generated PlantUML file can then be compared with [this one](https://github.com/LIonWeb-org/organization/blob/main/lioncore/metametamodel.puml): they should have exactly the same contents apart from some a couple of obvious differences.
+Run [`watch-tests.sh`](watch-tests.sh) to run all unit tests in watch mode.
 
 
-## Dev dependencies
+### Dev dependencies
 
 * [Deno](https://deno.land/): {Java|Type}Script runtime
-* (optional) [PlantUML](https://plantuml.com/)
+* (optional) [PlantUML](https://plantuml.com/).
+  An IDE plugin such as the one [for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/7017-plantuml-integration) also does the trick.
+
+
+## Getting started
+
+The following is a list of links to potential starting points:
+
+* Implementation of the LIonCore meta-metamodel (M3): see the [specific README](src/m3/README.md)
+* Metamodel-generic/-aspecific code regarding:
+  * [TypeScript type definitions](src/types.ts)
+  * (representation of) [references](src/references.ts)
+  * [serialization](src/serialization.ts)
+  * [generation of IDs](src/id-generation.ts)
+
+
+## Repository organization
+
+* [Diagrams](diagrams/) - various diagrams.
+  The PlantUML file [`diagrams/metametamodel-gen.puml`](diagrams/metametamodel-gen.puml) is generated from the meta-circular definition of `lioncore` in [`src/m3/test/self-definition.ts`](src/m3/test/self-definition.ts)
+  This generated PlantUML file can then be compared with [this one](https://github.com/LIonWeb-org/organization/blob/main/lioncore/metametamodel.puml): they should have exactly the same contents apart from some a couple of obvious differences.
```

Comment:
I think there is a typo around "some a couple"

Reply:
Thanks, fixed!